inventory/management/incoming/three_steps.rst

```diff
@@ -2,113 +2,108 @@
 Process a Receipt in three steps (Input + Quality + Stock)
 ==========================================================
 
-In many companies, it is necessary to assess the received good. The goal
-is to check that the products correspond to the quality requirements
-agreed with the suppliers. Therefore, adding a *quality control step*
-in the goods receipt process can become essential.
-
-Odoo uses routes to define exactly how you will handle the different
-receipt steps. The configuration is done at the level of the warehouse.
-By default, the reception is a one-step process, but changing the
-configuration can allow having 3 steps.
-
-The 3-steps flow is as follows: You receive the goods in an input area,
-then transfer them into a quality area for *quality control*. When the
-quality check has been processed, you can move the goods from QC to
-stock. Of course, you may change the quantity and only transfer to stock
-the quantity that is valid and decide that you will return the quantity
-that is not good.
-
-Multi-Step Routes
-=================
-
-First, you will need to activate the *multi-step routes* option.
-Indeed, routes provide a mechanism to chain different actions together.
-Here, we will chain the picking to the shipping.
-
-To activate *multi-step routes*, open the *inventory app*, and go to
-:menuselection:`Configuration --> Settings` and activate the option. By default,
-activating *multi-step routes* will also activate *Storage
-Locations*.
+In most companies, it is necessary to assess the received goods. The goal is to check that the 
+products correspond to the quality requirements agreed with the suppliers. Therefore, adding a 
+quality control step in the goods receipt process is essential.
+
+Odoo uses routes to define how to handle the different receipt steps.
+The configuration is done at the warehouse level. By default, the reception is a one-step process, 
+but it can also be configured to have two-steps or three-steps processes.
```

Comment:
Odoo uses routes to define how to handle the different receipt steps. Configuration of those routes is done at the warehouse level.